src/feed-input.scss

```diff
@@ -49,6 +51,35 @@ $block: #{$fd-namespace}-feed-input;
 
     &::placeholder {
       font-style: var(--fdFeed_Input_Placeholder_Font_Style);
+      color: #32363a;
+      color: var(--sapTextColor, #32363a);
+
+      &::-webkit-input-placeholder {
+        color: #32363a;
+        color: var(--sapTextColor, #32363a);
+      }
+
+      &::-moz-placeholder {
+        color: #32363a;
+        color: var(--sapTextColor, #32363a);
+      }
+
+      &:-ms-input-placeholder {
+        color: #32363a;
+        color: var(--sapTextColor, #32363a);
+      }
+
+      &:-moz-placeholder {
+        color: #32363a;
+        color: var(--sapTextColor, #32363a);
+      }
```

Comment:
- you don't need the fallback

Reply:
Do you really need `&::-webkit-input-placeholder`, `&::-moz-placeholder`, etc.? Have you tested without them?

Reply:
Fixed, removed since its working in the latest version of firefox and other browser